src/node/uvm_endorsements.h

```diff
@@ -63,7 +63,7 @@ namespace ccf
     {"did:x509:0:sha256:I__iuL25oXEVFdTP_aBLx_eT1RPHbCQ_ECBQfYZpt9s::eku:1.3.6."
      "1.4.1.311.76.59.1.5",
      "ConfAKS-AMD-UVM",
-     "0"}};
+     "1"}};
```

Comment:
@achamayou I support it's fine?

Also, am I right these shall be passed as a config to CCF instead?

Reply:
Reving up the SVN is fine. These are hardcoded for startup right now (see recent history under #6482), but can be updated later through governance. We probably want  to remove that initial hardcoding (see incomplete sub-item of #6482), but need to find a solution for JS API usage.